util/stop/stopper_test.go

```diff
@@ -274,16 +272,18 @@ func TestStopperNumTasks(t *testing.T) {
 				t.Fatalf("%d: expected %d tasks, got %d:\n%s", i, expNum, v, m)
 			}
 		}
-		task.Done()
+		// Close the channel to let the task proceed.
+		close(c)
+		// Wait for task completion.
+		time.Sleep(5 * time.Millisecond)
```

Comment:
we don't have a mechanism to guarantee that runPostlude() has executed. The sleep aims to achieve that.